packages/verified-fetch/test/get-stream-from-async-iterable.spec.ts

```diff
@@ -19,7 +20,7 @@ describe('getStreamFromAsyncIterable', () => {
     const chunks = new TextEncoder().encode('Hello, world!')
     const iterator = (async function * () { yield chunks })()
     const { firstChunk, stream } = await getStreamFromAsyncIterable(iterator, 'test.txt', defaultLogger(), { onProgress: onProgressSpy })
-    expect(firstChunk).to.equal(chunks)
+    expect(equals(firstChunk, chunks)).to.equal(true)
```

Comment:
You can use a simpler assertion:

```suggestion
    expect(firstChunk).to.equalBytes(chunks)
```